rclcpp_lifecycle/README.md

```diff
@@ -2,7 +2,7 @@
 
 Package containing a prototype for lifecycle implementation.
 
-Visit the [rclcpp_lifecycle API documentation](http://docs.ros2.org/latest/api/rclcpp_lifecycle/) for a complete list of its main components and features. For more information about LifeCycle in ROS 2, see the [design document](http://design.ros2.org/articles/node_lifecycle.html).
```

Comment:
Oopsies, I removed the link to the lifecycle nodes design document. Re-adding it.